search/indexing_api.py

```diff
@@ -190,6 +190,19 @@ def serialize_program_enrolled_user(program_enrollment):
     return serialized
 
 
+def filter_current_work(profile):
+    """
+    Remove work_history objects that are not current
+    """
+    current_employment = []
```

Comment:
It looks like we can use a list comprehension here instead, can you modify the function to return one?